masterlist.yaml

```diff
@@ -19681,11 +19681,11 @@ plugins:
   - name: 'dodge.esp'
     req:
       - name: '../ScriptDragon.dll'
-        display: '[Script Dragon](http://www.dev-c.com/?altname=skyrim_script_dragon)'
+        display: '[Script Dragon](https://www.dev-c.com/?altname=skyrim_script_dragon)'
   - name: 'Dual Wield Parrying.*\.esp'
     req:
       - name: '../ScriptDragon.dll'
-        display: '[Script Dragon](http://www.dev-c.com/?altname=skyrim_script_dragon)'
+        display: '[Script Dragon](https://www.dev-c.com/?altname=skyrim_script_dragon)'
```

Comment:
The website doesn't appear to actually support HTTPS. It's also dead; the current link is [here](http://www.dev-c.com/skyrim/scriptdragon/).